src/applications/personalization/profile/models/CommunicationChannel.js

```diff
@@ -1,6 +1,6 @@
 class CommunicationChannel {
   constructor({ type, parentItemId, permissionId, isAllowed, wasAllowed }) {
-    if (typeof type !== 'number' || type < 1 || type > 2) {
+    if (typeof type !== 'number' || type < 1 || type > 3) {
```

Comment:
I can only assume this was done as a kind of warning that you're misusing this component if the "type" is not in that `channelsWithContactInfo` set. But I think this is becoming cumbersome now that we're adding something that doesn't fit the historical `channelsWithContactInfo` set of 1 and 2. This might be a good area for a little refactoring. 

Also, the next guard starting on line 8 is not even reachable.

Reply:
I tried to limit my changes here and adjusted this value because our communication channel is `3`. I would like to get clarity on the significance of channels but agree we could refactor.